src/data/actions/filters.js

```diff
@@ -31,7 +31,7 @@ const initializeFilters = ({
     assignmentGradeMax,
     courseGradeMin,
     courseGradeMax,
-    includeCourseRoleMembers,
```

Comment:
What does this do?

Reply:
clears a complaint that the value is being passed from urlQuery as string rather than boolean.